internal/store/encryption_at_rest.go

```diff
@@ -18,19 +18,19 @@ import (
 	"fmt"
 
 	"github.com/mongodb/mongodb-atlas-cli/internal/config"
-	atlas "go.mongodb.org/atlas/mongodbatlas"
+	atlasv2 "go.mongodb.org/atlas/mongodbatlasv2"
 )
 
 //go:generate mockgen -destination=../mocks/mock_encryption_at_rest.go -package=mocks github.com/mongodb/mongodb-atlas-cli/internal/store EncryptionAtRestDescriber
 
 type EncryptionAtRestDescriber interface {
-	EncryptionAtRest(string) (*atlas.EncryptionAtRest, error)
+	EncryptionAtRest(string) (*atlasv2.EncryptionAtRest, error)
```

Comment:
Dependent on https://jira.mongodb.org/browse/CLOUDP-168455
EncryptionAtRest struct is missing GroupID

Reply:
Are there documentation/release note changes required to capture this?